### PR TITLE
Tom multiple legend

### PR DESCRIPTION
**Pull request by Derryrover**

### CODE REVIEW

File: app/components/legend/legend-service.js [hunk 1]
```diff
@@ -230,6 +231,8 @@ angular.module('legend')
           dataLayerObj = _.find(DataService.dataLayers, { uuid: uuid });
           if (!dataLayerObj) { return; }
           this.uuidMapping[uuid] = name;
+          var tempRaster = _.find(DataService.dataLayers, { uuid: uuid });
+          this.uuidOrganisationMapping[uuid] =  tempRaster && tempRaster.organisation && tempRaster.organisation.name;
```
Comment (GeOdin) [inline app/components/legend/legend-service.js]: Is it certain that you only need this.uuidOrganisationMapping[uuid] for temporal rasters?

File: app/components/legend/templates/legend.html [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
+        style="font-weight:lighter; font-size: 16px; color:white;"
+      >
+        LEGENDS
+      </span>
+    </div>
 
-  <div class="card legend-discrete">
+  <div 
+    class="card legend-discrete"
+    style="background-color:transparent;"
```
Comment (GeOdin) [inline app/components/legend/templates/legend.html]: Why does the background color has to be transparent here?

File: app/components/legend/templates/legend.html [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
           class="legend-header"
           style="margin-bottom:20px"
         >
-          <a href="#"
-             ng-click="switchSelectedRaster(uuid)"
-             title="{{ 'Switch selected legend' | translate }}">
+         
             {{ legend.uuidMapping[uuid] }}
+            {{ " - "}}
+            {{ uuidOrganisationMapping[uuid] }}
```
Comment (GeOdin) [inline app/components/legend/templates/legend.html]: NIce! :)

File: app/components/legend/templates/legend.html (hunk 2 above)
Comment (Derryrover) [reply app/components/legend/templates/legend.html]: Another class sets it to a certain color.
Then it falls over the dark background of the parent div.
But I want the the dark color of this background div to show.
Therefore I just overwrite the background color to be transparent.

File: app/components/legend/legend-service.js (hunk 1 above)
Comment (Derryrover) [reply app/components/legend/legend-service.js]: No it is not certain. We may need more details.
Anyway, this is how it is was done with the "this.uuidMapping[uuid] = name" for the names a few lines higher.
I decided to use the same approach.

There is probably a more elegant way to pass the entire raster object to the legend, but I got it working like this now. I may consider passing the entire object.

**Pull request merged**